core/data/src/main/java/com/dev/innerview/core/data/di/DataModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.dev.innerview.core.data.di
+
+import com.dev.innerview.core.data.repository.InnerViewRepositoryImpl
+import com.dev.innerview.core.data_api.InnerViewRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@Module
+@InstallIn(SingletonComponent::class)
+abstract class DataModule {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line
<!-- __reviewdog__:ChBjYzBlOTQ1MTU4OGZjMTc0EgZrdGxpbnQ= -->